mypy/stubgen.py

```diff
@@ -1077,7 +1077,7 @@ def visit_import_from(self, o: ImportFrom) -> None:
             self.record_name(alias or name)
 
         if self._all_:
-            # Include import froms that import names defined in __all__.
+            # Include import forms that import names defined in __all__.
```

Comment:
I think the original version was correct here. This is referring to "import froms" as in `from x import y` statements, not import forms.

Reply:
Nice catch!